lib/core/Canvas.js

```diff
@@ -190,6 +190,11 @@ export default function Canvas(config, eventBus, graphicsFactory, elementRegistr
    */
   this._rootElement = null;
 
+  /**
+   * @type {boolean}
+   */
+  this._focused = false;
```

Comment:
Let's also expose this as an API.

Reply:
Useful in the following scenario:

```
eventBus.on([ 'canvas.focusin', 'canvas.focusout' ], () => {
  if (canvas.isFocussed()) {
    // bind keyboard listeners
  } else {
    // unbind keyboard listeners
  }
});
```

Alternatively we can better support this via a more general event `canvas.focus.changed`:

```
eventBus.on([ 'canvas.focus.changed' ], (event) => {
  if (event.focussed) {
    // bind keyboard listeners
  } else {
    // unbind keyboard listeners
  }
});
```

Reply:
The later one will simplify things, and better align with `canvas.viewbox.changed` that we already emit.